heuristics/hyades_heuristic.py

```diff
@@ -0,0 +1,71 @@
+import os
+import numpy
+from cfmm_base import infotodict as cfmminfodict
+from cfmm_base import create_key
+
+def infotodict(seqinfo):
+    """Heuristic evaluator for determining which runs belong where
+    allowed template fields - follow python string module:
+    item: index within category
+    subject: participant id
+    seqitem: run number during scanning
+    subindex: sub index within group
+    """
+
+    # call cfmm for general labelling and get dictionary
+    info = cfmminfodict(seqinfo)
+
+    # key definitions
+    # anat
+    t1w = create_key('{bids_subject_session_dir}/anat/{bids_subject_session_prefix}_T1w')
+    flair  = create_key('{bids_subject_session_dir}/anat/{bids_subject_session_prefix}_FLAIR')
+    t2w = create_key('{bids_subject_session_dir}/anat/{bids_subject_session_prefix}_T2w')
+    
+    # dwi. added _acq-mde as a key to identify this as a MDE scan
+    dwi = create_key('{bids_subject_session_dir}/dwi/{bids_subject_session_prefix}_acq-mde_dir-AP_dwi')
+    # Save the RPE (reverse phase-encode) B0 image as a fieldmap (fmap).
+    fmap_rev_phase =  create_key('{bids_subject_session_dir}/fmap/{bids_subject_session_prefix}_dir-PA_epi')
+    
+    # MTS collection
+    mt_off = create_key('{bids_subject_session_dir}/anat/{bids_subject_session_prefix}_flip-1_mt-off_MTS')
+    mt_on  = create_key('{bids_subject_session_dir}/anat/{bids_subject_session_prefix}_flip-1_mt-on_MTS')
+    mt_t1w  = create_key('{bids_subject_session_dir}/anat/{bids_subject_session_prefix}_flip-2_mt-off_MTS')
+    
+    # Fill data dictionary
+    info[t1w]=[]
```

Comment:
The T1w, T2w, and FLAIR can be left out too - they should be picked up by the base heuristic that this inherits from:

```python
info[t1w] = []
info[flair] = []
info[t2w] = []
```

It may cause duplicate conversions otherwise. One quick way to check is to see if your converted outputs contain both `{bids_subject_session_prefix}_T1w` as well as `{bids_subject_session_prefix}_acq-MPR*_run-*_T1w`.

Reply:
Ooooooh I see! I'll take a look to `cfmminfodict` to check what its already there.

If I still want to add new values to the info dictionary (e.g. the labels fot MTS that I am planning to use) I can leave that part of the code (keys and criteria) as it is, correct? I just need to properly append them to the info dictionary.